### PR TITLE
Documentation/account creation

### PR DESCRIPTION
**Pull request by Steffengreiner**
**What was changed**

Provide Documentation for user registration and password reset with screenshots

**Why it's a draft**
The documentation text is done, however the screenshots are outdated and await the final design of the login page.

### CODE REVIEW

File: docs/user/user_registration.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+![login_page_registration](images/user_registration/login_page.png)
+
+You should now be able to see the registration site:
+
+![registration page](images/user_registration/registration_page.png)
+
+Please provide the required information which consists of: 
+
+1. Your full name consisting of your first name and surname
+2. Your email of choice which will be associated with your account
```
Comment (wow-such-code) [inline docs/user/user_registration.md:24]:
```suggestion
2. A valid email address of choice which will be associated with your account
```

File: docs/user/user_registration.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+4. A secure password consisting of at least 12 characters
+
+Once all the information has been provided correctly, 
+press the register button on the bottom to create your account. 
+
+## Validate the registered account
+
+Before you can log into the data manager platform with your newly created account, 
+you need to validate that the created account indeed belongs to the provided email address.
```
Comment (wow-such-code) [inline docs/user/user_registration.md:38]:
```suggestion
you need to validate the provided email address.
```

File: docs/user/user_registration.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+For this you will receive an email shortly after creating the account containing a validation link.
```
Comment (wow-such-code) [inline docs/user/user_registration.md:39]:
```suggestion
You will receive an email shortly after creating the account containing a validation link.
```

File: docs/user/password_reset.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+3. [Set a new password for the account according to the password policy](#set-a-new-password-for-the-account)
+
+## Trigger Password Reset
+
+From the login page you can navigate to a dedicated password reset site via the "Forgot password"
+link
+on the bottom of the login
+![login_password_reset](images/password_reset/password_reset.png)
+
+You should now be able to see the password reset site:
```
Comment (wow-such-code) [inline docs/user/password_reset.md:18]:
```suggestion
You should now be able to see the password reset dialog:
```

File: docs/user/password_reset.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+to set a new password should a password associated with a user account be lost.
+To reset your password the following steps have to be taken
+
+1. [Provide the account credentials for which the password should be reset](#trigger-password-reset)
+2. [Follow the password reset link you received in your email](#validate-the-password-reset-trigger)
+3. [Set a new password for the account according to the password policy](#set-a-new-password-for-the-account)
+
+## Trigger Password Reset
+
+From the login page you can navigate to a dedicated password reset site via the "Forgot password"
```
Comment (wow-such-code) [inline docs/user/password_reset.md:13]:
```suggestion
From the login page you can navigate to a dedicated password reset dialog via the "Forgot password"
```

File: docs/user/password_reset.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+link
+on the bottom of the login
+![login_password_reset](images/password_reset/password_reset.png)
+
+You should now be able to see the password reset site:
+
+![password_reset_email](images/password_reset/password_reset_email.png)
+
+Please provide the email address of the account for which the password should be reset
+and press the "Send" button which will email the provided account with further instructions
```
Comment (wow-such-code) [inline docs/user/password_reset.md:23]:
```suggestion
and press the "Send" button. You will receive an email with further instructions.
```

File: docs/download/create_pat.md [hunk 7]
```diff
@@ -35,6 +36,6 @@ so you remember for what purpose you have created them.
 
 ![token overview](images/token_overview.png)
 
-!!! warning
-    If you are unsure that your PAT got exposed or shared with untrusted parties, delete them right
+!!! danger "token security"
+    If you are unsure that your PAT got exposed or shared with untrusted parties, delete them right
```
Comment (wow-such-code) [inline docs/download/create_pat.md:40]:
```suggestion
    If you are unsure, if your PAT got exposed or shared with untrusted parties, delete them right 
```

**Review by wow-such-code — APPROVED** [review]
Correct and easy to understand. I made a few sentence suggestions, but feel free to ignore.

File: docs/user/password_reset.md (hunk 4 above)
Comment (Steffengreiner) [reply docs/user/password_reset.md:18]: Let's compromise on "view". A dialog is in our context a pop up window into which the user will provide his information

File: docs/user/password_reset.md (hunk 5 above)
Comment (Steffengreiner) [reply docs/user/password_reset.md:13]: Let's compromise on "view". A dialog is in our context a pop up window into which the user will provide his information

Comment (wow-such-code) [reply docs/user/password_reset.md]: agreed!

**Review by wow-such-code — APPROVED** [review]

**Pull request merged**